src/main/java/org/opensearch/ad/transport/ADBatchAnomalyResultRequest.java

```diff
@@ -70,8 +70,6 @@ public ActionRequestValidationException validate() {
         AnomalyDetector detector = adTask.getDetector();
         if (detector == null) {
             validationException = addValidationError("Detector can't be null", validationException);
-        } else if (detector.isRealTimeDetector()) {
-            validationException = addValidationError("Can't run batch task for realtime detector", validationException);
```

Comment:
Why remove this validation?

Reply:
This method is deprecated `isRealTimeDetector`. We don't differentiate realtime and historical detector from AD1.1.

Reply:
Oh right, I saw that was removed. Makes sense